### DISCUSSION TITLE
single selection per group?

### DISCUSSION THREAD

**Discussion by bonesoul** [JedWatson/react-select#5022] [Q&A]
I've setup some groups and child entries for them. What I want is to force user to only select one entry per group.

Is this possible?

**Comment by Rall3n** [comment]
Hello @bonesoul,

it is possible by using the `onChange` prop to update the state only if the new value is part of an unselected group.

The `onChange` prop provides the selected option in its second argument.

To make the comparison easier you could implement an identifying group key in each option.
